src/shared/sam/cli/samCliBuild.ts

```diff
@@ -55,6 +55,9 @@ export interface FileFunctions {
     fileExists: typeof fileExists
 }
 
+/**
+ * An elaborate way to run `sam build`.
```

Comment:
Is this (and other) comments necessary? Thinking this is implied by the function name/file structure

Reply:
none of this was obvious to me.